### PR TITLE
Fix/arrows

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-319

#### Description
This PR makes the usage of arrows on cards consistent throughout.

#### Screenshot of the result
n/a

#### Additional comments or questions
[Sibling PR in the design system](https://github.com/danskernesdigitalebibliotek/dpl-design-system/pull/412)

### CODE REVIEW

**Comment by JacobArrow** [comment]
I'm waiting for the implementation in design system before approving this :)

File: src/apps/dashboard/dashboard-notification/dashboard-notification.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const DashboardNotification: FC<DashboardNotificationProps> = ({
       onClick={notificationClickEvent}
       className="mb-16"
     >
-      <div className="list-dashboard shadow-medium-hover arrow__hover--right-small">
+      <div className="list-dashboard shadow-medium-hover arrow arrow__hover--right-small">
```
Comment (JacobArrow) [inline src/apps/dashboard/dashboard-notification/dashboard-notification.tsx:34]: Where is the `arrow` class in design system? Cant find it😬

File: src/components/Buttons/ArrowButton.tsx [hunk 2]
```diff
@@ -18,6 +18,7 @@ const ArrowButton: React.FC<ArrowButtonProps> = ({
   return (
     <button
       aria-labelledby={arrowLabelledBy}
+      className="arrow-button"
```
Comment (JacobArrow) [inline src/components/Buttons/ArrowButton.tsx:21]: Why does this not have `arrow__hover--right-small` like the rest?

**Review by JacobArrow — DISMISSED** [review]
I had some thoughts

**Review by JacobArrow — CHANGES_REQUESTED** [review]
I had some thoughts

File: src/apps/dashboard/dashboard-notification/dashboard-notification.tsx (hunk 1 above)
Comment (Adamik10) [reply src/apps/dashboard/dashboard-notification/dashboard-notification.tsx:34]: I'm not sure whether it's there, but I believer we use the arrow class together whenever we use arrow__hover--right-small

File: src/components/Buttons/ArrowButton.tsx (hunk 2 above)
Comment (Adamik10) [reply src/components/Buttons/ArrowButton.tsx:21]: What do you mean like the rest? the arrow__hover--right-small is from my understanding only used on the card element containing the arrow inside it.

Comment (JacobArrow) [reply src/components/Buttons/ArrowButton.tsx:21]: Just sharing my thoughts, and wanted to make sure it was not an oversight. Carry on good sir 😁

**Review by JacobArrow — APPROVED** [review]
Let's gooooo 🚀

**Pull request merged**